src/main/java/co/rsk/bitcoinj/core/StoredBlock.java

```diff
@@ -35,11 +35,19 @@
  */
 public class StoredBlock {
 
-    // A BigInteger representing the total amount of work done so far on this chain. As of May 2011 it takes 8
-    // bytes to represent this field, so 12 bytes should be plenty for now.
-    public static final int CHAIN_WORK_BYTES = 12;
-    public static final byte[] EMPTY_BYTES = new byte[CHAIN_WORK_BYTES];
-    public static final int COMPACT_SERIALIZED_SIZE = BtcBlock.HEADER_SIZE + CHAIN_WORK_BYTES + 4;  // for height
+    /* A BigInteger representing the total amount of work done so far on this chain. As of June 22, 2024, it takes 12
+     unsigned bytes to store this value, so developers should use the V2 format. */
+    private static final int CHAIN_WORK_BYTES_V1 = 12;
+    // A BigInteger representing the total amount of work done so far on this chain.
```

Comment:
I think this comment may be misplaced or simply wrong. It's talking about the chainwork, not the chainwork size in bytes right?

Reply:
You are right. The way it is written is confusing. What is trying to express is what this size of bytes in `int` is representing.

I will update it to improve it.

Reply:
Updated